sphinxcontrib/adadomain.py

```diff
@@ -510,7 +510,7 @@ class AdaDomain(Domain):
         "function": AdaObject,
         "procedure": AdaObject,
         "type": AdaObject,
-        "setpackage": AdaSetPackage,
+        "set_package": AdaSetPackage,
```

Comment:
The README should be updated accordingly

(what about merging this into the first commit btw?)

Reply:
It was a mess because actually we reuse the name "package" for a directive in the next commit. So I merged the two commits, it's much clearer now